db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :hostname
```

Comment:
Ok, so based on the [comment in the other thread](https://github.com/ManageIQ/manageiq/pull/7996#discussion-diff-60134968), this should definitely be `host_name`...it does not represent a URL-like hostname (as far as I can tell).